nimare/generate.py

```diff
@@ -44,7 +44,7 @@ def create_coordinate_dataset(
         Number of foci considered to be noise in each study. (Default=0)
     seed : :obj:`int`
         Random state to reproducibly initialize random numbers.
```

Comment:
Perhaps add something like "If None is provided, then the random state will be randomly selected."